src/main/kotlin/org/opensearch/commons/alerting/model/Monitor.kt

```diff
@@ -175,6 +177,7 @@ data class Monitor(
         builder.field(DATA_SOURCES_FIELD, dataSources)
         builder.field(DELETE_QUERY_INDEX_IN_EVERY_RUN_FIELD, deleteQueryIndexInEveryRun)
         builder.field(SHOULD_CREATE_SINGLE_ALERT_FOR_FINDINGS_FIELD, shouldCreateSingleAlertForFindings)
+        builder.field(FAN_OUT_ENABLED_FIELD, fanoutEnabled)
```

Comment:
If we decide to keep this attribute at the monitor level instead of moving it to the doc level monitor input class, we should probably only include the field in the `toXContent` output for doc level monitors to avoid confusion.